features/org.palladiosimulator.simulizar.test.commons.feature/feature.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.palladiosimulator.simulizar.test.commons.feature"
+      label="Palladio Component Model - Analyzer: SimuLizar - Test Commons"
+      version="5.0.0.qualifier"
+      plugin="org.palladiosimulator.branding"
+      license-feature="org.palladiosimulator.license"
+      license-feature-version="1.0.0">
+
+   <description url="http://www.example.com/description">
+      [Enter Feature Description here.]
+   </description>
+
+   <requires>
+      <import feature="org.palladiosimulator.simulizar.feature" version="5.0.0" match="compatible"/>
+      <import plugin="org.palladiosimulator.pcm" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="org.palladiosimulator.simulizar" version="5.0.0" match="greaterOrEqual"/>
+      <import plugin="org.palladiosimulator.analyzer.workflow"/>
+      <import plugin="org.junit.jupiter.api"/>
+      <import plugin="de.uka.ipd.sdq.workflow.mdsd" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="de.uka.ipd.sdq.simucomframework" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="de.uka.ipd.sdq.simulation.abstractsimengine"/>
+      <import plugin="de.uka.ipd.sdq.simulation.abstractsimengine.desmoj"/>
+      <import plugin="org.palladiosimulator.recorderframework.edp2" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="org.palladiosimulator.edp2"/>
+      <import plugin="de.uka.ipd.sdq.codegen.simucontroller" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="com.google.guava"/>
+      <import plugin="org.eclipse.xtext.xbase.lib"/>
+      <import plugin="org.eclipse.xtend.lib"/>
+      <import plugin="org.eclipse.xtend.lib.macro"/>
+      <import plugin="org.palladiosimulator.edp2.util" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="de.uka.ipd.sdq.simucomframework.variables" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="org.palladiosimulator.probeframework" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="org.palladiosimulator.recorderframework" version="4.3.0" match="greaterOrEqual"/>
+      <import plugin="com.google.dagger"/>
+      <import plugin="org.hamcrest"/>
```

Comment:
We need to make sure, that the version is at least 2.2.0. Otherwise, it could pick up the plugin from Orbit, which is likely to fail, as the hamcrest library is missing.